packages/browser/src/backend.ts

```diff
@@ -132,18 +132,15 @@ export class BrowserBackend implements Backend {
    * @inheritDoc
    */
   public async sendEvent(event: SentryEvent): Promise<SentryResponse> {
-    let dsn: DSN;
-
     if (!this.options.dsn) {
-      throw new SentryError('Cannot sendEvent without a valid DSN');
-    } else {
-      dsn = new DSN(this.options.dsn);
+      // We do nothing in case there is no DSN
+      return { status: Status.Skipped };
```

Comment:
It'd be nice to provide a `reason` why we skip some of the events and use `logger` to let user know about it